src/openrct2/world/ConstructionClearance.h

```diff
@@ -22,6 +22,16 @@ class QuarterTile;
 
 using CLEAR_FUNC = int32_t (*)(TileElement** tile_element, const CoordsXY& coords, uint8_t flags, money64* price);
 
+// Used when calling MapCanConstructWithClearAt();
+// This assumes that the caller has already done the check on the element it wants to place,
+// as this function can only check the element the player wants to build through.
+enum class CreateCrossingMode
+{
+    none,
+    trackOverPath,
+    pathOverTrack,
```

Comment:
Shouldn't it be TitleCase? I'm confused as to whether this changed recently
```suggestion
    None,
    TrackOverPath,
    PathOverTrack,
```

Reply:
It was recently changed to `lowerCamelCase`, in line with how OpenLoco does it.